core/arch/arm/kernel/link.mk

```diff
@@ -30,6 +30,7 @@ entries-unpaged += sm_init
 entries-unpaged += core_init_mmu_regs
 entries-unpaged += sem_cpu_sync
 entries-unpaged += generic_boot_get_handlers
+entries-unpaged += plat_cpu_reset_late
```

Comment:
Actually using `KEEP_PAGER(plat_cpu_reset_late)` fails: `plat_cpu_reset_late` is linked inside the 'int' sections, not the 'unpaged' section. I must find out why.